docs/data/material/getting-started/templates/checkout/PaymentForm.tsx

```diff
@@ -1,65 +1,250 @@
+              borderRadius: '20px',
+              border: '2px solid ',
+              borderColor: 'divider',
+              backgroundColor: 'background.paper',
+              boxShadow: '0px 4px 8px rgba(0, 0, 0, 0.05)',
+            }}
+          >
+            <Box sx={{ display: 'flex', justifyContent: 'space-between' }}>
+              <Typography variant="subtitle2">Credit card</Typography>
+              <CreditCardRoundedIcon sx={{ color: 'text.secondary' }} />
+            </Box>
+            <SimCardRoundedIcon
+              sx={{
+                fontSize: { xs: '48px', sm: '56px' },
+                transform: 'rotate(90deg)',
+                color: 'text.secondary',
+              }}
+            />
+            <Box
+              sx={{
+                display: 'flex',
+                justifyContent: 'space-between',
+                width: '100%',
+                gap: 3,
+              }}
+            >
+              <FormGrid sx={{ flexGrow: 1 }}>
+                <FormLabel required>Card number</FormLabel>
+                <OutlinedInput
+                  id="standard-basic"
+                  autoComplete="card-number"
+                  placeholder="0000 0000 0000 0000"
+                  required
+                  value={cardNumber}
+                  onChange={handleCardNumberChange}
+                />
+              </FormGrid>
+              <FormGrid sx={{ maxWidth: '20%' }}>
+                <FormLabel required>CVV</FormLabel>
+                <OutlinedInput
+                  id="cvv"
+                  autoComplete="CVV"
+                  placeholder="123"
+                  required
+                  value={cvv}
+                  onChange={handleCvvChange}
+                />
+              </FormGrid>
+            </Box>
+            <Box sx={{ display: 'flex', justifyContent: 'space-between', gap: 3 }}>
+              <FormGrid sx={{ flexGrow: 1 }}>
+                <FormLabel required>Name</FormLabel>
+                <OutlinedInput
+                  id="card-name"
+                  autoComplete="card-name"
+                  placeholder="John Smith"
+                  required
+                />
+              </FormGrid>
+              <FormGrid sx={{ flexGrow: 1 }}>
+                <FormLabel required>Expiration date</FormLabel>
+                <OutlinedInput
+                  id="card-expiration"
+                  autoComplete="card-expiration"
+                  placeholder="MM/YY"
+                  required
+                  value={expirationDate}
+                  onChange={handleExpirationDateChange}
+                />
+              </FormGrid>
+            </Box>
+          </Box>
+        </Box>
+      )}
+      {paymentType === 'creditCard' && (
+        <FormControlLabel
+          control={<Checkbox name="saveCard" />}
+          label="Remember credit card details for next time"
+          sx={{ justifyContent: 'center' }}
+        />
+      )}
```

Comment:
```suggestion

```

We're duplicating the conditionals here — we can delete it from here and add it on the key above